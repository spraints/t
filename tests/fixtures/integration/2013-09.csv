2013-09-02 08:00,2013-09-02 09:43
2013-09-02 13:08,2013-09-02 13:41
2013-09-02 14:28,2013-09-02 14:39
2013-09-02 15:59,2013-09-02 16:12
2013-09-02 16:20,2013-09-02 19:58
2013-09-03 07:15,2013-09-03 07:39
2013-09-03 08:15,2013-09-03 10:03
2013-09-03 12:21,2013-09-03 13:00
2013-09-03 13:30,2013-09-03 13:59
2013-09-03 15:30,2013-09-03 16:54
2013-09-03 17:13,2013-09-03 17:36
2013-09-03 17:41,2013-09-03 17:48
2013-09-03 18:21,2013-09-03 18:55
2013-09-03 19:14,2013-09-03 20:05
2013-09-04 07:40,2013-09-04 09:12
2013-09-04 09:54,2013-09-04 10:12
2013-09-04 10:13,2013-09-04 11:43
2013-09-04 11:55,2013-09-04 12:03
2013-09-04 12:19,2013-09-04 13:09
2013-09-04 15:00,2013-09-04 18:55
2013-09-05 07:34,2013-09-05 10:14
2013-09-05 13:45,2013-09-05 19:07
2013-09-06 14:05,2013-09-06 14:20
2013-09-07 15:25,2013-09-07 16:38
2013-09-09 07:04,2013-09-09 07:31
2013-09-09 07:44,2013-09-09 08:45
2013-09-09 10:02,2013-09-09 11:14
2013-09-09 11:15,2013-09-09 11:21
2013-09-09 11:35,2013-09-09 13:17
2013-09-09 13:38,2013-09-09 14:07
2013-09-09 14:58,2013-09-09 17:08
2013-09-09 21:31,2013-09-09 22:25
2013-09-10 07:09,2013-09-10 08:09
2013-09-10 08:44,2013-09-10 09:49
2013-09-10 10:00,2013-09-10 10:40
2013-09-10 11:01,2013-09-10 11:04
2013-09-10 11:40,2013-09-10 11:55
2013-09-10 14:00,2013-09-10 16:22
2013-09-10 21:59,2013-09-10 22:10
2013-09-11 07:46,2013-09-11 12:31
2013-09-11 13:14,2013-09-11 16:07
2013-09-11 16:50,2013-09-11 17:06
2013-09-11 18:26,2013-09-11 20:56
2013-09-12 08:03,2013-09-12 08:59
2013-09-12 13:48,2013-09-12 16:25
2013-09-12 16:50,2013-09-12 17:14
2013-09-12 18:45,2013-09-12 20:03
2013-09-13 09:07,2013-09-13 12:18
2013-09-13 14:01,2013-09-13 14:18
2013-09-14 13:00,2013-09-14 13:43
2013-09-16 08:11,2013-09-16 09:46
2013-09-16 10:33,2013-09-16 12:08
2013-09-16 15:33,2013-09-16 15:58
2013-09-16 16:05,2013-09-16 16:27
2013-09-16 19:53,2013-09-16 20:38
2013-09-16 21:03,2013-09-17 00:13
2013-09-17 08:21,2013-09-17 08:25
2013-09-17 08:55,2013-09-17 10:44
2013-09-17 11:09,2013-09-17 12:18
2013-09-17 16:09,2013-09-17 16:15
2013-09-17 16:40,2013-09-17 17:13
2013-09-17 18:20,2013-09-17 18:30
2013-09-17 22:02,2013-09-18 01:21
2013-09-18 09:08,2013-09-18 13:43
2013-09-18 14:27,2013-09-18 15:20
2013-09-18 16:45,2013-09-18 19:16
2013-09-19 07:26,2013-09-19 08:50
2013-09-19 10:11,2013-09-19 10:46
2013-09-19 13:34,2013-09-19 18:00
2013-09-19 18:25,2013-09-19 20:40
2013-09-20 08:15,2013-09-20 08:49
2013-09-20 08:49,2013-09-20 09:09
2013-09-20 09:56,2013-09-20 11:31
2013-09-20 12:42,2013-09-20 13:12
2013-09-20 13:51,2013-09-20 14:25
2013-09-21 06:25,2013-09-21 07:07
2013-09-23 06:29,2013-09-23 08:53
2013-09-23 10:15,2013-09-23 10:49
2013-09-23 12:24,2013-09-23 15:09
2013-09-23 17:24,2013-09-23 19:58
2013-09-23 21:04,2013-09-23 21:56
2013-09-24 07:23,2013-09-24 07:38
2013-09-24 08:06,2013-09-24 08:38
2013-09-24 08:51,2013-09-24 09:08
2013-09-24 10:18,2013-09-24 10:25
2013-09-24 11:14,2013-09-24 11:36
2013-09-24 12:09,2013-09-24 15:00
2013-09-24 15:33,2013-09-24 16:02
2013-09-24 16:12,2013-09-24 16:16
2013-09-24 16:22,2013-09-24 19:31
2013-09-25 07:22,2013-09-25 07:46
2013-09-25 09:18,2013-09-25 12:47
2013-09-25 13:33,2013-09-25 16:45
2013-09-25 16:59,2013-09-25 17:07
2013-09-26 07:29,2013-09-26 09:08
2013-09-26 13:26,2013-09-26 13:43
2013-09-26 15:09,2013-09-26 15:22
2013-09-26 15:46,2013-09-26 18:58
2013-09-27 08:52,2013-09-27 11:58
2013-09-28 04:50,2013-09-28 05:58
2013-09-28 21:12,2013-09-28 21:20
2013-09-29 21:32,2013-09-29 22:00
2013-09-30 09:06,2013-09-30 14:07
2013-09-30 15:46,2013-09-30 18:03
