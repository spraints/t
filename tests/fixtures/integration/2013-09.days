2013-09-01 - 2013-09-07 ||       |   378 |   399 |   493 |   482 |    15 |    73 ||   1840
2013-09-08 - 2013-09-14 ||       |   481 |   336 |   624 |   315 |   208 |    43 ||   2007
2013-09-15 - 2013-09-21 ||       |   459 |   362 |   560 |   520 |   213 |    42 ||   2156
2013-09-22 - 2013-09-28 ||       |   549 |   486 |   433 |   321 |   186 |    76 ||   2051
2013-09-29 - 2013-10-05 ||    28 |   438 |       |       |       |       |       ||    466
2013-09                 ||    28 |  2305 |  1583 |  2110 |  1638 |   622 |   234 ||   8520
2013                    ||    28 |  2305 |  1583 |  2110 |  1638 |   622 |   234 ||   8520
8h=480m 16h=960m 24h=1440m 32h=1920m 40h=2400m
