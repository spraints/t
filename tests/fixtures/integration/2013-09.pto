2013-09-01 work=1840 pto= 560
2013-09-08 work=2007 pto= 393
2013-09-15 work=2156 pto= 244
2013-09-22 work=2051 pto= 349
2013-09-29 work= 466 pto=1934

2013 total_pto= 3480 days=  7
8h=480m 16h=960m 24h=1440m 32h=1920m 40h=2400m
