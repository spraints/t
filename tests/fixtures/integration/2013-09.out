2013-09-01 - 2013-09-07   1840 min   24 segments  min/avg/max/stddev=  7/ 76/322/ 81  ▃▁▁▁▅  ▁▃▁▁▂▁▁▁▂  ▃▁▂▁▂▆  ▄▇  ▁  ▂
2013-09-08 - 2013-09-14   2007 min   26 segments  min/avg/max/stddev=  3/ 77/285/ 70  ▁▂▂▁▃▁▄▂  ▂▂▁▁▁▄▁  ▇▅▁▄  ▂▄▁▂  ▅▁  ▂
2013-09-15 - 2013-09-21   2156 min   28 segments  min/avg/max/stddev=  4/ 77/275/ 71  ▃▃▁▁▂▅  ▁▁▃▂▁▁▁▄  ▃▇▂▄  ▃▁▇▄  ▁▁▃▁▁  ▂
2013-09-22 - 2013-09-28   2051 min   25 segments  min/avg/max/stddev=  4/ 82/209/ 77  ▅▂▆▆▂  ▁▂▁▁▁▆▁▁▇  ▁▇▇▁  ▄▁▁▇  ▇  ▃▁
2013-09-29 - 2013-10-05    466 min    3 segments  min/avg/max/stddev= 28/155/301/137  ▁  ▇▄
8h=480m 16h=960m 24h=1440m 32h=1920m 40h=2400m
